frontend/javascripts/viewer/view/left-border-tabs/layer_settings_tab.tsx

```diff
@@ -23,6 +23,7 @@ import {
   updateDatasetDefaultConfiguration,
 } from "admin/rest_api";
 import { Button, Col, Divider, Dropdown, type MenuProps, Modal, Row, Switch } from "antd";
+import type { SwitchChangeEventHandler } from "antd/es/switch";
```

Comment:
I vaguely remember that these path imports caused trouble with antd :thinking: weren't the problems theming related then? can the import path be changed `antd`?